canonical-web-and-design/charmed-osm.com
# Update and Switch over to github actions

## Done

Replace Circle CI with Github actions

## QA

Github action checks should turn green:
 lint-python
 lint-scss
 run-dotrun
 run-image
 test-python

## Issue / Card

Testing sprint
